scripts/sct_download_data.py

```diff
@@ -115,7 +116,8 @@ def main(args=None):
         'deepseg_lesion_models': ['https://osf.io/eg7v9/?action=download',
                               'https://www.neuro.polymtl.ca/_media/downloads/sct/20180613_deepseg_lesion_models.zip'],
         'c2c3_disc_models': ['https://osf.io/t97ap/?action=download',
-                            'https://www.neuro.polymtl.ca/_media/downloads/sct/20190117_c2c3_disc_models.zip']
+                            'https://www.neuro.polymtl.ca/_media/downloads/sct/20190117_c2c3_disc_models.zip'],
+        'modality_pred_model': ['https://www.neuro.polymtl.ca/_media/downloads/sct/20190706_modality_pred_model.pt.zip']
```

Comment:
I have not checked on the wiki yet, but: Did you put your model into a folder? or did you directly zip the model?

Reply:
(I would recommend to add the model into a folder and to zip the folder)

Reply:
Ok I can do that! Why though ?

Reply:
just to follow the convention of other models, which are enclosed into function-specific folder under **data/** (e.g., deepseg_gm_models/, deepseg_lesion_models/, gm_model/, etc.)